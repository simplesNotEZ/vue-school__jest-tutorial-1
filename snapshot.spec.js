const user = {
  name: 'Tony Tinkertons',
  age: 42,
  job: 'inventor'
};

describe('compare/contrast snapshot testing', () => {
  // NOT as a snapshot
  test('userString as manaully-created string matches user', () => {
    const userString = '{"name":"Tony Tinkertons","age":42,"job":"inventor"}';
    expect(JSON.stringify(user)).toEqual(userString);
  });

  // Yes! as a snapshot
  test('user matches snapshot', () => {
    expect(user).toMatchSnapshot();
  });
});
