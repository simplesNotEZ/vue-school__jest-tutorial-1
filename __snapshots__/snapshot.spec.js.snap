// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compare/contrast snapshot testing user matches 1`] = `
Object {
  "age": 42,
  "job": "inventor",
  "name": "Tony Tinkertons",
}
`;
